src/GenerateOfflineMapExe.cpp

```diff
@@ -147,6 +155,10 @@ int main(int argc, char** argv) {
 		CommandLineDouble(dFillValueOverride, "fillvalue", 0.0);
 		CommandLineBool(fInputConcave, "in_concave");
 		CommandLineBool(fOutputConcave, "out_concave");
+		CommandLineDouble(lb, "lb", 0.0);
```

Comment:
The default arguments for these bounds look arbitrary. Or are these scaled versions of some bounds that apply to all variables in general ?